server/routes/index.js

```diff
@@ -33,4 +34,12 @@ app.get('/api/levels', levelsService.list);
 app.get('/api/levels/:id', levelsService.findById);
 app.put('/api/levels/:id', levelsService.update);
 
+// notes endpoints
+app.post('/api/incidents/:id', notesService.create);
```

Comment:
Could we have this endpoint specifying on the route that its POSTing a note?`/api/incidents/:id/notes`